code/nebula/neb.cpp

```diff
@@ -332,6 +332,19 @@ void neb2_init()
 	parse_modular_table("*-neb.tbm", parse_nebula_table);
 }
 
+// set the bits for poofs from a list of poof names
+void neb2_set_poof_bits(SCP_vector<SCP_string> list)
+{
+	Neb2_poof_flags = 0; //Make absolutely sure flags are zero'd before we start adding to it-Mjn
+	for (const SCP_string& thisPoof : list) {
+		for (int i = 0; i < (int)Poof_info.size(); i++) {
+			if (Poof_info[i].name == thisPoof) {
```

Comment:
I'd recommend a case-insensitive comparison, since most mission parsing checks are case-insensitive.
```suggestion
			if (SCP_string_lcase_equal_to()(Poof_info[i].name, thisPoof)) {
```